### ISSUE TITLE
Breaking change: v8.2.0 returns only file name in path property, not the path

### ISSUE THREAD

**Issue by barbalex** [react-dropzone/react-dropzone#769]
- [x] I found a bug
- [ ] I want to propose a feature

**What is the current behavior?**
The files Objects returned on `onDrop` used to return the entire file path in the `path` property. Sinse v8.2.0 not any more.

**What is the expected behavior?**
The `path` property should contain the entire path, just as it does in v8.1.0.
Returning only the file name is an undeclared breaking change. And it makes no sense as the `path` property now returns exactly the same data as the `name` property.

Our usage of Dropzone depends on the entire path being returned: Our app only saves the full path as url. It does nothing with the files themselves.

**Comment by rolandjitsu** [comment]
@barbalex this feel like an issue with [file-selector](https://github.com/react-dropzone/file-selector) and not with dropzone. But I have a feeling it has been resolved in https://github.com/react-dropzone/file-selector/commit/e5ecce836a4bc8a51059fe21426d1290bb5d2bb8.

If you think the issue is still unresolved, please open an issue there.

**Comment by rolandjitsu** [comment]
Right now it's set to [`^0.1.8`](https://github.com/react-dropzone/react-dropzone/blob/master/package.json#L85), so it should pick up on the latest fixes if you run the install command again (make sure to ignore the package lock files).

But it will be updated in a next version.

**Comment by barbalex** [comment]
If I get this correctly https://github.com/react-dropzone/file-selector/commit/e5ecce836a4bc8a51059fe21426d1290bb5d2bb8 was published with v1.9.0 6 days ago. I updated react-dropzone yesterday (repeatedly, using v0.8.2 and v0.9.0) and it did not output full paths. 

So it seems to me that there is additional action necessary from users - beyond just updating - to prevent updating to react-dropzone v0.8.2 or 0.9.0 from breaking behaviour.

In other words: quickly publishing a new version might save a bunch of other users from running into issues.

**Comment by barbalex** [comment]
This is still an issue in v10.0.4. Need to remain on v8.1.0

**Comment by barbalex** [comment]
```
$ yarn list --pattern file-selector
yarn list v1.13.0
└─ file-selector@0.1.11
Done in 3.56s.
```
So it seems that https://github.com/react-dropzone/file-selector/commit/e5ecce836a4bc8a51059fe21426d1290bb5d2bb8 was not the solution

**Comment by barbalex** [comment]
Also I now do not believe any more that file-selector is the reason as I have tried:

```
  "resolutions": {
    "file-selector": "0.1.6"
  }
```
...and the error still occurs (also with all versions up to 0.1.11)

**Comment by barbalex** [comment]
Correction: I just realized that according to my yarn.lock file react-dropzone v8.1.0 had no dependency on file-selector. So it seems that file-selector _is_ the reason for the issue.

**Comment by Shaker-Hamdi** [comment]
@barbalex I have the same issue, how did you manage to solve it.
This is my package-lock.json ...

```
"react-dropzone": {
      "version": "10.1.9",
      "resolved": "https://registry.npmjs.org/react-dropzone/-/react-dropzone-10.1.9.tgz",
      "integrity": "sha512-7iqALZ0mzk+4g/AsYxEy3QyWPMTVQYKQVkYUe9zIbH18u+pi7EBDg010KEwfIX6jeTDH2qP0E6/eUnXvBYrovA==",
      "requires": {
        "attr-accept": "1.1.3",
        "file-selector": "0.1.12",
        "prop-types": "15.7.2"
      }
    }
```

**Comment by barbalex** [comment]
@Shaker-Hamdi  It is not solved. I am still stuck on v8.1.0 :-(

The reason this issue is closed is that the issue seems to be in file-selector, see here: https://github.com/react-dropzone/file-selector/issues/10

**Comment by barbalex** [comment]
I am consistently getting this behavior in versions 11 to 14 again when using tauri. See: https://github.com/react-dropzone/file-selector/issues/10#issuecomment-1170136347
